openmetadata-ui/src/main/resources/ui/src/components/MyData/MyData.component.tsx

```diff
@@ -134,10 +134,9 @@ const MyData: React.FC<MyDataProps> = ({
                 {ownedData.length ? (
                   <Link
                     data-testid="my-data"
-                    to={getExploreLinkByFilter(
-                      Ownership.OWNER,
-                      AppState.userDetails,
-                      AppState.nonSecureUserDetails
+                    to={getUserPath(
+                      AppState.getCurrentUserDetails()?.name || '',
```

Comment:
Can we store this value in a variable and use it everywhere?

```
const currentUser = AppState.getCurrentUserDetails();
```
```suggestion
                      currentUser?.name || '',
```

Reply:
Already tried this, wasn't working correctly